### ISSUE TITLE
Name Suggestion Index V6 - LIVE

### ISSUE THREAD

**Issue by UKChris-osm** [osmlab/name-suggestion-index#4964]
Update: Version 6 is now live via the newly released 2.20.1 version of the iD Editor. Simply logging in to OpenStreetMap and clicked edit should now open up the newest version of iD with the freshest version of the NSI :)

Thank you to @bhousel for the hard work, and to everyone who has contributed to version 5 and 6 over the last year :)

= = = = = = = = = = = = = = = = = = = = = = 

Version 6 of the Name Suggestion Index has been submitted and accepted into the iD Editor (openstreetmap/iD#8305) by @bhousel, and should hopefully make it into the new 2.20.0 version of the iD Editor's release.

A preview of this is available at: [https://ideditor.org/nsi-v5](https://ideditor.org/nsi-v5)

**Comment by bhousel** [comment]
Yep!  as mentioned on #4543 we able to publish v5 releases now, and the test iD can pull from them
Like all releases - it took way longer than I expected, and didn't include as much as I'd like!  😆 

I'd love for more people to test here and let me know if you notice any issues:
https://ideditor.org/nsi-v5

Another thing I did recently - I moved a lot of the project documentation to the GitHub wiki:
https://github.com/osmlab/name-suggestion-index/wiki

So I think that means contributors can edit this - it would be great to have more people look it over and make less confusing!
